### PR TITLE
PIX minimun value warning

### PR DESCRIPTION
**Pull request by malkafly-cw**
Added a warning when the payment amount is less than BRL 1.00 in the PIX

image

### CODE REVIEW

**Review by panachi-cw — APPROVED** [review]
42

**Review by bru-carenzi — CHANGES_REQUESTED** [review]
### Test Summary

**Status**:  👎 Not recommended 

---


📋 Test report 
Insert here which tests you have done in this PR and put a ✅ [Check Mark Button](https://emojipedia.org/check-mark-button/) emoji for those tests that are validated, or put the [❌ Cross Mark](https://emojipedia.org/cross-mark/) for the tests with error.

* Amount less than 1,00 should appears the warning - ✅ 
* Amount == 1,00 should not appears the warning - ❌
* Amount > 1,00 should not appears the warning - ✅ 

🙆‍♀️ Non-block issues
# 

🙅 Block issues
It's not been accepting amount == 1,00  to pay with Pix.

File: src/Controller/Checkout.php [hunk 1]
```diff
@@ -224,6 +224,12 @@ public function process_pix() {
 			$order_value = $orderTotalWithDiscount * 100;
 			$final_value = (int) explode( '.', $order_value )[0];
 
+			if( $final_value < 101 ) {
```
[the remark this answers is not in the document]
Comment (malkafly-cw) [reply src/Controller/Checkout.php:227]: Oh! I misunderstood the rule, I thought it was above 1, I will fix it now. Thanks!

**Review by bru-carenzi — APPROVED** [review]
### Test Summary

**Status**: ready 

---


📋 Test report 
Insert here which tests you have done in this PR and put a ✅ [Check Mark Button](https://emojipedia.org/check-mark-button/) emoji for those tests that are validated, or put the [❌ Cross Mark](https://emojipedia.org/cross-mark/) for the tests with error.

Pix amount < 1,00: 
* Warning when the amount is < 1,00 - ✅ 
* Warning does not show up when the amount is >= 1,00 - ✅ 
* Pix Payment - ✅

**Review by panachi-cw — APPROVED** [review]

**Review by bru-carenzi — COMMENTED** [review]
@malkafly-cw could u implement character limits in the CPF field to 11 numbers, pls?
Also, we need to apply the item 3 for this field as well: 3. Campos numéricos de cartão devem der o type number (exibir teclado no mobile)
From a mobile device the CPF field accept letters and number and has not limit:

[CHECK THE IMAGE](https://user-images.githubusercontent.com/93997860/205922112-9f32782e-f32e-4b98-b16e-33fcff67befe.png)

Pedidos PIX não mudam de status na loja, mesmo após ser aprovado na InfinitePay

In the tests the order after be approved change the status to "Processando" and when this order has not payment, the status is "Pagemento pendente". The same happens to credit card payment, this is the expect behavior? cc: @malkafly-cw @panachi-cw

Also, we need to fix this problem: https://github.com/cloudwalk/product-ecommerce/issues/67

**Review by bru-carenzi — CHANGES_REQUESTED** [review]
### Test Summary

**Status**:  👎 Not recommended 

---

**Comment by malkafly-cw** [comment]
@bru-carenzi the css, numeric field on CPF and log pix webhook are implemented and ready to test

**Review by panachi-cw — APPROVED** [review]
Update the changelog

**Pull request merged**